doc/users/next_whats_new/begginarrow_endarrow.rst

```diff
@@ -0,0 +1,5 @@
+String values for ``ArrowStyle._Curve.endarrow`` and ``ArrowStyle._Curve.beginarrow``
```

Comment:
This should refer only to the public API, e.g., how it affects `ArrowStyle` or `ConnectionPatch` or anywhere that the string is exposed directly.